app/controllers/microservices_engine/v1/data_controller.rb

```diff
@@ -42,11 +42,11 @@ def update
         data = params['content']
         if data
```

Comment:
Hashes do not support `.present?` (not in my ruby 2.3 irb at least)

A missing build or token will fail the request (I can put `.require` back but I was tentative because of my story in that router PR)